src/seedu/addressbook/AddressBook.java

```diff
@@ -830,11 +830,11 @@ private static void deletePersonFromAddressBook(int index) {
 	 * @return true if the given person was found and deleted in the model
 	 */
 	private static boolean deletePersonFromAddressBook(String[] exactPerson) {
```

Comment:
Okay now I see, I misuse the coding standard which says that "boolean values should always start with 'is'"